107/src/main/java/org/ehcache/jsr107/ServiceConfigurationParser.java

```diff
@@ -50,7 +50,7 @@ public URI getNamespace() {
   }
 
   @Override
-  public ServiceConfiguration<Jsr107Service> parse(final Element fragment) {
+  public ServiceCreationConfiguration<Jsr107Service> parse(final Element fragment) {
```

Comment:
`JSr107ServiceConfigurationParser` seems better. I will rename though the XML parser interfaces to indicate better what they stand for.